client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts

```diff
@@ -405,7 +405,13 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
     public async onStopButton(speaker: ViewSpeaker): Promise<void> {
         try {
             if (speaker.speech_state !== SpeechState.INTERPOSED_QUESTION && this.interposedQuestions.length > 0) {
-                this.onPauseButton(speaker);
+                if (speaker.isSpeaking) {
+                    this.onPauseButton(speaker);
+                } else {
+                    this.interposedQuestions
+                        .filter(speaker => speaker.isSpeaking)
+                        .forEach(speaker => this.onPauseButton(speaker));
+                }
```

Comment:
I'm not exactly sure why this is not awaited. 

```suggestion
                if (speaker.isSpeaking) {
                    await this.onPauseButton(speaker);
                } else {
                    for (let q of this.interposedQuestions.filter(speaker => speaker.isSpeaking)) {
                        await this.onPauseButton(speaker);
                    }
                }
```

Reply:
Good call, I restructured the loop a bit